golint.toml

```diff
@@ -4,7 +4,7 @@ errorCode = 2
 
 #TODO we should eventually comment all public methods
```

Comment:
You can remove that TODO as well. Also, maybe we remove that entire entry instead of setting "Disabled=false"?